ChatApp/Core/CustomUI/TypingAreaView.swift

```diff
@@ -62,6 +68,14 @@ class TypingAreaView: UIView {
         layer.cornerRadius = Constants.cornerRadius
     }
     
+    func messageTextViewColorConfigure(_ isDarkMode: Bool) {
+        if isDarkMode {
+            messageTextView.textColor = .white
+        } else {
+            messageTextView.textColor = .black
```

Comment:
არ არის აუცილებელი მაგრამ ტერნარით შეგეძლო კოდის შემცირება
```suggestion
        messageTextView.textColor = isDarkMode ? .white :  .black
```